tls/s2n_config.h

```diff
@@ -182,3 +182,4 @@ int s2n_config_free_session_ticket_keys(struct s2n_config *config);
 void s2n_wipe_static_configs(void);
 extern struct s2n_cert_chain_and_key *s2n_config_get_single_default_cert(struct s2n_config *config);
 int s2n_config_get_num_default_certs(struct s2n_config *config);
+S2N_RESULT s2n_config_wall_clock(struct s2n_config *config, uint64_t *output);
```

Comment:
Your issue mentions `monotonic_clock `. Do we also want a wrapper for that?

Reply:
Since there are only 2 occurrence of monotonic_clock callbacks, the wrapper can be ignored. @lrstewart , correct ?

Reply:
Agreed. The timer struct / methods are the wrapper already.